static/app/components/events/autofix/autofixTimeline.tsx

```diff
@@ -69,27 +66,33 @@ export function AutofixTimeline({events}: Props) {
     <Timeline.Container>
       {events.map((event, index) => {
         const isActive = event.is_most_important_event && index !== events.length - 1;
+        const isTruncated = event.isTruncated;
+
         return (
           <Timeline.Item
             key={index}
             title={
               <StyledTimelineHeader
-                onClick={() => toggleItem(index)}
+                onClick={isTruncated ? undefined : () => toggleItem(index)}
                 isActive={isActive}
+                isTruncated={isTruncated}
                 data-test-id={`autofix-root-cause-timeline-item-${index}`}
               >
                 {event.title}
-                <StyledIconChevron
-                  direction={expandedItems.includes(index) ? 'down' : 'right'}
-                  size="xs"
-                />
+                {!isTruncated && (
+                  <StyledIconChevron
+                    direction={expandedItems.includes(index) ? 'down' : 'right'}
+                    size="xs"
+                  />
+                )}
               </StyledTimelineHeader>
             }
-            icon={getEventIcon(event.timeline_item_type)}
-            colorConfig={getEventColor(isActive)}
+            isActive={isActive}
```

Comment:
small nit: I prefer how it looks without the circle, it's not even a circle it's an oval...i assume `isActive` here is what triggers it to show?

Reply:
I find it much more readable with the circle. `isActive` is the trigger. I'll see if I can fix the stretching though

Reply:
Ok I fixed the stretching, but I still wanna keep the circle: 
<img width="542" alt="Screenshot 2025-02-05 at 11 40 36 AM" src="https://github.com/user-attachments/assets/36cb051a-a683-405a-af9a-e5c086d9b196" />